install/npmManager.go

```diff
@@ -0,0 +1,56 @@
+package install
+
+import (
+	"bytes"
+	"os/exec"
+	"regexp"
+)
+
+// NpmManager is a manager for nodejs
+type NpmManager struct{}
+
+// Install function of NpmManager installs npm dependencies
+func (m NpmManager) Install(c Context, r Requirement) string {
+	globalFlag := ""
+	if c.Environment == "global" {
+		globalFlag = "-g"
+	}
+	packageFlag := r.Package
+	if len(r.Version) > 0 {
+		packageFlag += "@" + r.Version
+	}
+	cmd := exec.Command(r.Manager, "install", globalFlag, packageFlag)
+	if len(c.Project) > 0 {
+		cmd.Dir = c.Project
+	}
+
+	var stderr bytes.Buffer
```

Comment:
I still suggest executing the install command outside on the lvl upper, so it will return string. Then it will allow to save this string in a file (for example for generating docker file).